install/kubernetes/helm/istio/values.yaml

```diff
@@ -87,6 +87,13 @@ kiali:
 istio_cni:
   enabled: false
 
+#
```

Comment:
please revert changes to this file.  The values.yaml are inherited from the subcharts subdir.

Reply:
I added this here only for documentation purposes. Since the `telemetry-gateway` subchart doesn't have a single `enabled` value I included both `*Enabled` here to show how to enable the addon. The values.yaml from the subchart itself includes more options.
Should I completely remove it or something like `telemetry-gateway: {}` would be okay for documenting the addon?

Reply:
It is documented here:

https://github.com/istio/istio/blob/master/install/kubernetes/helm/subcharts/telemetry-gateway/values.yaml#L1-L6

Work is also in progress to produce from the subcharts a list of KV pairs in this issue tracker:
https://github.com/istio/istio.io/issues/2735

https://github.com/istio/istio/pull/9106

and this PR: https://github.com/istio/istio.io/pull/2784

We would welcome your help if you like 👍

Reply:
Thank you for pointing these out! I'll see where I can help. :) I reverted the changes to this file, the PR only includes `requirements.yaml` now.